t/t0052-simple-ipc.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+test_description='simple command server'
+
+. ./test-lib.sh
+
+test-tool simple-ipc SUPPORTS_SIMPLE_IPC || {
+	skip_all='simple IPC not supported on this platform'
+	test_done
+}
```

Comment:
Should this be a prereq defined in `test-lib.sh`? That way we can require SUPPORTS_SIMPLE_IPC when we eventually make our fsmonitor tests.

Reply:
> Should this be a prereq defined in `test-lib.sh`?

I originally had this as a prereq, but since _all_ of the tests would need that prereq, I opted for the `skip_all` instead.

> That way we can require SUPPORTS_SIMPLE_IPC when we eventually make our fsmonitor tests.

That's not actually necessary, as there is _another_ prereq for the FSMonitor: not all platforms that support simple IPC support FSMonitor. Besides, the fact that FSMonitor uses simple IPC is an implementation detail that should not bleed into `t/t7527-internal-fsmonitor.sh`.